PowerShell.Common.props

```diff
@@ -95,7 +95,7 @@
     <Company>Microsoft Corporation</Company>
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '127.0.0.1', but got 10.79.8.80.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 258
258:             $result[0].TargetAddress | Should -BeExactly $realAddress
```